tools/GenerateTestSummary/Program.cs

```diff
@@ -44,14 +51,21 @@
         if (Directory.Exists(dirPathOrTrxFilePath))
         {
             var trxFiles = Directory.EnumerateFiles(dirPathOrTrxFilePath, "*.trx", SearchOption.AllDirectories);
-            foreach (var trxFile in trxFiles)
+            if (!trxFiles.Any())
```

Comment:
`Directory.EnumerateFiles` returns an `IEnumerable`. Does this make it iterate over the filesystem the second time?

Reply:
Changed this to use a list.